std/algorithm/setops.d

```diff
@@ -611,6 +593,27 @@ void largestPartialIntersection
             sorted);
 }
 
+///
```

Comment:
As this edit might be a bit controversial I moved it over to #4066, so it doesn't block this PR ;-)